releasenotes/config.yaml

```diff
@@ -37,11 +37,14 @@ template: |
       Add normal bug fixes here, or remove this section.
 
 sections:
-  # The prelude section is implicitly included.
+  # The highlights section is implicitly included.
   - [upgrade, Upgrade Notes]
   - [features, New Features]
   - [enhancements, Enhancement Notes]
   - [issues, Known Issues]
   - [deprecations, Deprecation Notes]
   - [security, Security Notes]
   - [fixes, Bug Fixes]
+  # DO NOT REMOVE. The following sections are no longer used, but were used in the past. Keeping them here avoids reno linting errors.
+  - [prelude, prelude]
+  - [preview, preview]
```

Comment:
`reno lint` raises errors when it encounters unknown sections.
We need to list these legacy sections here.
The alternative would be fixing old release notes, which would be incorrectly included in the next release.